README.md

```diff
@@ -182,7 +182,7 @@ class Raven_Client
 ```json
     "extra": {
         "branch-alias": {
-            "dev-master": "1.10.x-dev"
+            "dev-master": "1.11.x-dev"
```

Comment:
On the section above this one the referenced version has been updated to `2.x.x` but here it's still `1.x.x`